packages/expo-localization/src/Localization.types.ts

```diff
@@ -65,3 +65,134 @@ export type Localization = {
    */
   timezone: string;
 };
+
+export type Locale = {
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) with a region code.
+   * @example `'en-US'`, `'es-419'`, `'pl-PL'`.
+   */
+  languageTag: string;
+  /**
+   * An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag) without the region code.
+   * @example `'en'`, `'es'`, `'pl'`.
+   */
+  languageCode: string;
+  /**
+   * The region code for your device that comes from the Region setting under Language & Region on iOS, Region settings on android and is parsed from locale on web (can be null on web).
+   */
+  regionCode: string | null;
+  /**
+   * Currency code for the locale.
+   * @example `'USD'`, `'EUR'`, `'PLN'`.
+   * Returns `null` on web, use a table lookup based on region instead.
+   */
+  currencyCode: string | null;
+  /**
+   * Currency symbol for the locale.
+   * @example `'$'`, `'€'`, `'zł'`.
+   * Returns `null` on web, use a table lookup based on region (if available) instead.
+   */
+  currencySymbol: string | null;
+  /**
+   * Decimal separator used for formatting numbers with fractional parts.
+   * @example `'.'`, `','`.
+   */
+  decimalSeparator: string | null;
+  /**
+   * Digit grouping separator used for formatting large numbers.
+   * @example `'.'`, `','`.
+   */
+  digitGroupingSeparator: string | null;
+  /**
+   * Text direction for the locale. One of: `'ltr'`, `'rtl'`, but can also be null on older browsers without support for the textInfo API.
+   */
+  textDirection: 'ltr' | 'rtl' | null;
+  /**
+   * The measurement system used in the locale. On iOS is one of `'metric'`, `'us'`. On android is one of `'metric'`, `'us'`, `'uk'`.
+   * Returns `null` on web, as user chosen measurement system is not exposed on the web and using locale to determine measurement systems is unreliable.
+   * Ask for user preferences if possible.
+   */
+  measurementSystem: `metric` | `us` | `uk` | null;
+};
+
+export enum Weekday {
```

Comment:
Could you add a doc comment for this enum? I guess it looks a bit bare in the docs when both the enum and its cases are undocumented 🤔